systemd/kano-desktop-uixinit.service

```diff
@@ -0,0 +1,25 @@
+#
+# kano-desktop-uixinit.service
+#
+# Starts the Desktop UIX Init service
+#
+# The "BindsTo" clause makes a behaviour dependency with the main service.
+# It is started and stopped along with the Desktop unit.
+#
+# FIXME: For now it's still under early testing
+#
+
+[Unit]
+Description=Kano Desktop setup script
+BindsTo=kano-desktop.service
+Requires=kano-desktop-boards-daemon.service kano-desktop-mouse.service \
+ kano-desktop-notifications.service kano-desktop-tracker-watch.service \
+ kano-desktop-feedback-widget.service kano-desktop-mount-trigger.service \
+ kano-desktop-world-widget.service kano-desktop-speakerleds.service \
+ kano-desktop-vnc.service
+
```

Comment:
I think some of these we want running in both dashboard and desktop mode, right? 
* notifications because we were asked for badges to still show up when apps are launched from desktop
* boards-daemon because it is needed for make-light
Also, I think we want to only run the tracking stuff once?
Similarly, I think the mouse is jsut doing xset and probably only needs to run once.

So these could either be added as Requires to kano-dashboard as well (which I think measn they won't get restarted) or  started from kano-uixinit-systemd. The former might be a bit faster as it would start them in parallel. If you do that, but don't want kano-dashboard to know about all these targets, then you could make another intermediate target.